irods-iBridgesCli.py

```diff
@@ -162,23 +169,28 @@ def prepareUpload(dataPath, ic, config):
             print('Aborted: Data path not given')
             return False
     else:
-        pass 
+        pass
+    #store verified dataPath
+    config["iRODS"]["uploadItem"] = dataPath 
 
     size = get_local_size([dataPath])
-    freeSpace = ic.get_resource(config['iRODS']['irodsresc']).free_space
-
-    print('Checking storage capacity for '+dataPath+', '+str(float(size)/(1024**3))+'GB')
-
-    if int(freeSpace)-1024**3 < size:
+    freeSpace = int(ic.get_free_space(config['iRODS']['irodsresc']))
+    print('Checking storage capacity for '+dataPath+', '+str(float(size)/(1000**3))+'GB')
```

Comment:
Keeping the units calculation in a single location is desirable for both performance and ease of maintenance.  I have tried to do this with the `MULTIPLIER` constant in `IrodsConnector.py`  It can be updated as desired.